### PR TITLE
fix: missing buttons lifting

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-1126](https://emurgo.atlassian.net/browse/YOMO-1126) and [YOMO-1142](https://emurgo.atlassian.net/browse/YOMO-1142)

[YOMO-1126]: https://emurgo.atlassian.net/browse/YOMO-1126?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1142]: https://emurgo.atlassian.net/browse/YOMO-1142?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jorbuedo — APPROVED** [review]

**Review by michalszorad — APPROVED** [review]

**Comment by michalszorad** [comment]
@stackchain please check the milestone

File: apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx [hunk 1]
```diff
@@ -51,60 +52,55 @@ export const ConfirmTxScreen = () => {
...
-
-      <View style={{paddingTop: 16, paddingHorizontal: 16}}>
-        <Fees yoroiUnsignedTx={yoroiUnsignedTx} />
-
-        <Spacer height={4} />
-
-        <BalanceAfter yoroiUnsignedTx={yoroiUnsignedTx} />
-      </View>
-
-      <ScrollView
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx:70]: the scroll bar flash was removed, was this intentional ? (it's inconsistent right now, not all scrolls have it) please add a task to check with Product, whether its added as default behaviour or we can drop it.

**Comment by stackchain** [comment]
> @stackchain please check the milestone

Including it.

File: apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx (hunk 1 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx:70]: Hi @stackchain 

This screen was copied from the send flow and if im not wrong here is not needed that scroll.

Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx:70]: I will confirm with product

Comment (banklesss) [reply apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx:70]: @stackchain 

in send flow the tx summary is dynamic. Here is not. So it doesn`t make sense to have that scroll here.

**Pull request merged**